runner/run.go

```diff
@@ -379,16 +379,26 @@ func getRequestFromTest(testRequest test.Input) *ftwhttp.Request {
 
 // applyInputOverride will check if config had global overrides and write that into the test.
 func applyInputOverride(o config.FTWTestOverride, testRequest *test.Input) error {
-	overrides := o.Input
+	overrides := o.Overrides
 	if overrides.Port != nil {
 		testRequest.Port = overrides.Port
 	}
+
+	if overrides.Headers != nil {
```

Comment:
I added headers override support starting from @snake66's [PR](https://github.com/coreruleset/go-ftw/pull/144) (Thank you!). Specifically speaking about the Host header, the individual header override will take precedence over a host header specified in the test and over the OverrideEmptyHostHeader feature. I added a test about it.